metadata-models/src/main/pegasus/com/linkedin/metadata/key/NotebookKey.pdl

```diff
@@ -0,0 +1,24 @@
+namespace com.linkedin.metadata.key
+
+/**
+ * Key for a Notebook
+ */
+@Aspect = {
+  "name": "notebookKey",
+}
+record NotebookKey {
+  /**
+  * The name of the Notebook tool such as QueryBook, etc.
+  */
+  @Searchable = {
+    "fieldName": "tool",
+    "fieldType": "TEXT_PARTIAL",
+    "boostScore": 4.0
+  }
+  notebookTool: string
```

Comment:
@dexter-mh-lee  I would like to consult what's the best way to deal with this information.

Reply:
Hey. I would actually suggest ingesting them as data platforms like urn:li:dataPlatform:mysql. This makes it easy for you to show logos to distinguish different platforms this notebook is based on, and also easily show these as filters in search. To do so, please include this aspect https://github.com/datahub-project/datahub/blob/master/metadata-models/src/main/pegasus/com/linkedin/common/DataPlatformInstance.pdl into the entity-registry.yml entry for this new entity, and make sure to ingest this aspect when writing the python connector!

For namespacing purposes, I would still keep this one here as string, but remove the Searchable annotation. You can see from the above link that DataPlatformInstance already has a field platform that is searchable and has addToFilters set to true! We could to discuss further if you have any qs

Reply:
Do you mean?
1. add `dataPlatformInstance` aspect to Notebook entity
2. rename this `notebookTool ` to something like `platform` with type string. But while ingesting the data, put platformUrn in this field

Reply:
1. Yes!
2. Leave this as is, and just remove the @Searchable annotation. 
When ingesting data, you will want to ingest this as string (like querybook) but also ingest the DataPlatformInstance aspect which has field platform, which is an URN.

Reply:
kk, got it. Changed it accordingly.